docs/docs/Configuration/configuration-authentication.md

```diff
@@ -103,12 +104,70 @@ LANGFLOW_SUPERUSER_PASSWORD=securepassword
 
 ### LANGFLOW_SECRET_KEY
 
-This environment variable holds a secret key used for encrypting the superuser's password. Make sure to set this to a secure, randomly generated string.
+This environment variable holds a secret key used for encrypting the superuser's password.
```

Comment:
It's not just the superuser's password though  - it's the key used for all secrets?